src/datachannel.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Data Channel</title>
+  <link rel="stylesheet" href="../style.css" />
+  <script src="../vcs-realtime-sdk.js"></script>
+  <script src="../config.js"></script>
+</head>
+
+<body>
+  <h2>Data Channel</h2>
+  <h3>Description</h3>
+  <p>This sample app joins a room with audio and video and shows all participants video along with the ability to send messages.</p>
```

Comment:
This description should focus on the sending of messages, not the audio/video part. So let's try to rewrite this a bit. E.g.

This sample app joins a room and then allow sending/receiving messages via a WebRTC Data Channel.